Support Velero 1.4 (#981)
SUSE/doc-caasp: adoc/admin-velero-backup.adoc

** Location
+

> Pass `--storage-location` to specify where the backup to store. For example, if we have an HA object storage server called default and secondary respectively.

Reviewer: ```suggestion
Pass `--storage-location` to specify where to store the backup. 
For example, if we have an HA object storage server called default and secondary respectively.
```

+
[source,bash]
----
# Create a backup to the default storage server
velero backup create backup2default --storage-location default

# Create a backup to the secondary storage server
velero backup create backup2secondary --storage-location secondary
----

** Garbage Collection
+

> Pass `--ttl` to specify how long the backup should be kept, after that, the backup will be dumped into the garbage (deleted). The default time for a backup to existing before it is deleted is 720 hours (30 days).

Reviewer: ```suggestion
Pass `--ttl` to specify how long the backup should be kept. After the specified time the backup will be deleted. 
The default time for a backup before deletion is 720 hours (30 days).
```
